automates/text_reading/src/main/scala/org/clulab/aske/automates/OdinActions.scala

```diff
@@ -1030,7 +1099,7 @@ a method for handling `ConjDescription`s - descriptions that were found with a s
 
   def combineFunction(mentions: Seq[Mention], state: State = new State()): Seq[Mention] = {
     val (functions, other) = mentions.partition(_.label == "Function")
-    val (complete, fragment) = functions.partition(m => m.arguments("input").nonEmpty && m.arguments("output").nonEmpty)
+    val (complete, fragment) = functions.partition(m => m.arguments.getOrElse("input", Seq()).nonEmpty && m.arguments.getOrElse("output", Seq()).nonEmpty)
```

Comment:
Might be able to do Seq.empty instead of Seq().

Reply:
Is it better to use Seq.empty instead of Seq()? What's the difference between the two?

Reply:
I changed the ones (`Seq()`) in combineFunction to Seq.empty, but I noticed there are more `Seq()` in other methods - Would it be better to replace them all with Seq.empty?

Reply:
Yep